solr/core/build.gradle

```diff
@@ -87,6 +87,7 @@ dependencies {
   runtimeOnly "org.apache.lucene:lucene-analysis-phonetic"
   runtimeOnly "org.apache.lucene:lucene-backward-codecs"
   implementation "org.apache.lucene:lucene-codecs"
+  implementation "org.apache.lucene:lucene-backward-codecs"
```

Comment:
necessary?

Reply:
To use the lucene90 codec it is necessary to add this dependency